picard/ui/options/general.py

```diff
@@ -73,7 +74,7 @@ class GeneralOptionsPage(OptionsPage):
         BoolOption("setting", "check_for_updates", True),
         IntOption("setting", "update_check_days", 7),
         IntOption("setting", "update_level", 0),
-        IntOption("persist", "last_update_check", 0),
+        IntOption("persist", "last_update_check", DEFAULT_PROGRAM_UPDATE_LEVEL),
```

Comment:
Shouldn't the `DEFAULT_PROGRAM_UPDATE_LEVEL` constant be applied to the "update_level" setting instead?

Reply:
Ooops, you're right of course, fixed and rebased.